code/_onclick/item_attack.dm

```diff
@@ -90,7 +100,7 @@
 	if(I.attack_verb && I.attack_verb.len)
 		message_verb = "[pick(I.attack_verb)]"
 	else if(!I.force)
-		return 0
+		return

```

Comment:
The return value of that proc is no longer used for anything. I think leaving a "return 0" could confuse readers thinking the "0" value has some significance.